### PR TITLE
Use standard module locations

### PR DESCRIPTION
**Pull request by grawity**
This allows the module to be automatically loaded by various programs.

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -4,21 +4,31 @@ _build=00161
...
-package() {
-  ar xf "ccs_$_version-$_build.amd64.deb"
-  tar xfz "data.tar.gz"
+prepare() {
+  bsdtar -xf data.tar.gz
+}
 
+package() {
   install -D -m644 LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
   install -D -m755 "usr/lib/ccs/libccpkip11.so.$pkgver" "${pkgdir}/usr/lib/libccpkip11.so"
```
Comment (2m) [inline PKGBUILD:26]: This is probably not needed anymore as added lines below do the same thing, right?

Comment (2m) [reply PKGBUILD:26]: Ahh, no. I see now. This one is under the `ccs` directory. Why do we need it in both paths?

File: PKGBUILD [hunk 2]
```diff
@@ -4,21 +4,31 @@ _build=00161
...
+
+  # Add to default module search path for various programs
+  install -d "${pkgdir}/usr/lib/pkcs11"
+  ln -s ../libccpkip11.so "${pkgdir}/usr/lib/pkcs11/libccpkip11.so"
+
+  # Support automatic loading in p11-kit (pkcs11.conf(5))
+  install -D -m644 ccpkip11.module "${pkgdir}/usr/share/p11-kit/modules/ccpkip11.module"
```
Comment (2m) [inline PKGBUILD:33]: Nice. What programs do you use LT ID card with that use `p11-kit`?

**Comment by 2m** [comment]
Thanks for the PR! Just a couple of questions. :)

File: PKGBUILD (hunk 1 above)
Comment (grawity) [reply PKGBUILD:26]: It was under ccs/ in the source, not the final package.

In general, all programs accept absolute paths anyway; most of them will look in either /usr/lib or /usr/lib/pkcs11 if just a name is provided.

(See also e.g. the whitelist in ssh-agent's source)

Comment (2m) [reply PKGBUILD:26]: Ahh, sorry. I meant `pkcs11`.

File: PKGBUILD (hunk 2 above)
Comment (grawity) [reply PKGBUILD:33]: None personally, but overall it's a rapidly growing list. (Even programs which aren't linked against it directly, e.g. wpa_supplicant and everything else that uses libp11/engine_pkcs11, will default to loading the `p11-kit-proxy` module – which then loads everything that p11-kit knows.)

**Comment by 2m** [comment]
Merged and pushed to AUR.

**Pull request merged**